### PR TITLE
Payment

### PR DESCRIPTION
**Pull request by loemhuolay**
Video Unlock(payment)
- Card Number
- Expire Date
- CVC Number

For Video unlock: <`http://localhost:3000/payments/new`>

### CODE REVIEW

File: pj_movie/app/views/payments/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <form action="#">
+  <h1>Video Unlock (payment)</h1>
+  <%= form_for("#") do |f| %>
+  <hr/><br/>
+  <%= f.label :card_number, "Card Number" %>
+  <%= f.text_field :text, class: 'info_field', placeholder: 'xxxx-xxxx-xxxx-xxxx' %>
+
+  <div class="info_field">
+  <%= f.label :expire_date, "Expire Date" %>
+  <input type="date" name="bday">
```
Comment (N310) [inline pj_movie/app/views/payments/new.html.erb]: the format of expire date is `MM/YYYY`.
date is unneeded.

File: pj_movie/app/assets/stylesheets/payments.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.payment{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/payments.scss]: You should add a space.
ex.
`.payment{` → `.payment {`

File: pj_movie/app/assets/stylesheets/static_pages.scss [hunk 3]
```diff
@@ -1,4 +1,3 @@
 // Place all the styles related to the StaticPages controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (PechBunsinh) [inline pj_movie/app/assets/stylesheets/static_pages.scss]: you should remove this code if you don't need

File: pj_movie/app/assets/javascripts/payment.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+      },
+      invalid: {
+        iconColor: '#000',
+        color: 'red',
+      },
+    },
+  });
+  card.mount('#bank_card');
+
+  registerElements([card], 'example1');
```
Comment (N310) [inline pj_movie/app/assets/javascripts/payment.js]: Now you can't use this function.
It's OK for now, but you have to confirm and fix when you start coding.

File: pj_movie/Gemfile [hunk 5]
```diff
@@ -8,9 +8,13 @@ end
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.1.6'
-# Use sqlite3 as the database for Active Record
+
 gem 'sqlite3'
 
+group :development do
+  gem 'rails_layout'
+end
```
Comment (N310) [inline pj_movie/Gemfile]: Where do you use this Gem?

File: pj_movie/app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,9 +1,13 @@
...
 <html>
   <head>
-  <title><%= yield(:title) %> | Project Movie</title>
+  <title><%= yield(:title) %>Project Movie</title>
   <%= render 'layouts/rails_default' %>
   <%= render 'layouts/shim' %>
+  <meta name="description" content="<%= content_for?(:description) ? yield(:description) : "Learn Rails" %>">
+    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
+    <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+    <%= csrf_meta_tags %>
```
Comment (N310) [inline pj_movie/app/views/layouts/application.html.erb]: Why do you add this lines?

File: pj_movie/Gemfile (hunk 5 above)
Comment (loemhuolay) [reply pj_movie/Gemfile]: I want to install Viewport but it not work

File: pj_movie/app/views/layouts/application.html.erb (hunk 6 above)
Comment (loemhuolay) [reply pj_movie/app/views/layouts/application.html.erb]: I research from http://railsapps.github.io/rails-default-application-layout.html it have these line to support Viewport.

File: pj_movie/app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %>Project Movie</title>
-    <%= render 'layouts/rails_default' %>
-    <%= render 'layouts/shim' %>
+  <title><%= yield(:title) %>Project Movie</title>
+  <%= render 'layouts/rails_default' %>
+  <%= render 'layouts/shim' %>
```
Comment (N310) [inline pj_movie/app/views/layouts/application.html.erb]: You shouldn't remove this empty.

**Pull request merged**